lib/api/v3/work_packages/schema/work_package_schema.rb

```diff
@@ -63,19 +59,19 @@ def assignable_statuses_for(user)
           end
 
           def assignable_versions
-            @work_package.assignable_versions if defines_assignable_values?
+            @work_package.assignable_versions unless @work_package.nil?
```

Comment:
For patterns like this you can use:

`@work_package.try(:assignable_versions)`